docs/modules/ROOT/pages/how-tos/upgrade-9.x-to-10.x.adoc

```diff
@@ -0,0 +1,44 @@
+= Upgrade from v9 to v10
+
+This guide describes the steps to perform an upgrade of the component from version v9 to v10.
+
+== Parameter changes
+
+* `charts.keycloakx` changed from `1.3.2` to `1.6.0`, the Keycloak image is updated from `17.0.2` to `18.0.2`.
+* `charts.postgresql` changed from `10.16.2` to `11.6.15`, the Postgresql version remains the same version `11.14.0-debian-10-r28`.
```

Comment:
did we override the postgres version in the old chart with version 11? this statement means, that no update of postgresql version is performed? 

The default is set to version 10:
https://github.com/projectsyn/component-keycloak/blob/v9.0.3/class/defaults.yml#L44

the only place 11.14 is configured in component version 9 is:
https://github.com/projectsyn/component-keycloak/blob/v9.0.3/tests/golden/builtin/builtin/builtin/01_keycloak_helmchart/postgresql/templates/statefulset.yaml#L101

The versioning seems to be a bit obfuscated but I assume, this statement is correct.

Reply:
reminder: chart version != postgresql image version.
Charts have their own versioning. A version update in the charts doesn't necessarily mean that the *image* within the chart is changed. It's entirely possible that the chart may change completely (e.g. parameter structure) and be a breaking/major change while the actual rendered deployment didn't change at all.